api.model.js

```diff
@@ -0,0 +1,11 @@
+const db = require('./db/connection')
+
+exports.topicFinder = () => {
```

Comment:
It is more conventional to name functions as verbs e.g. findTopics/fetchTopics/readTopics :)